index.html

```diff
@@ -0,0 +1,334 @@
+			</div>
+			<a class="btn-mediaConnect" href="#">LET’S CONNECT</a>
+			<ul class="media-list">
+				<li>
+					<a href="" class="media-link"><img src="./images/twitterIcon.svg" alt="twitterIcon" class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/linkedIncon.svg" alt="linkdlnIcon" class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/mailIcon.svg" alt="mailIcon" class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/githubIcon.svg" alt="githubIcon" class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/inconito.svg" alt="inconitoIcon" class="media-icon"></a>
+				</li>
+			</ul>
+		</main>
+
+		<!-- Partie 2 Project Part for review-->
+
+		<main id="section-project">
+			<div class="details-card">
+				<div class="project-image"><img src="./images/project_1.svg" alt="project1img"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Tonic</h1>
+							<div class="client-info">
+								<p class="canopy">CANOPY</p>
+								<ul class="role">
+									<li>Back End Dev</li>
+									<li>2015</li>
+								</ul>
+							</div>
+						</div>
+						<div class="project-info">
+							<p>
+								A daily selection of privately personalized reads;
+								no accounts or sign-ups required.
+							</p>
+						</div>
+						<div class="project-tag">
+							<ul class="lang">
+								<li>html</li>
+								<li>css</li>
+								<li>javascript</li>
+							</ul>
+						</div>
+					</div>
+					<div class="btn-project">
+						<a href="">See Project</a>
+					</div>
+				</div>
+			</div>
+			<div class="details-card">
+				<div class="project-image"><img src="./images/project2.svg" alt="projet1img"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Multi-Post Stories</h1>
+							<div class="client-info">
+								<p class="canopy">CANOPY</p>
+								<ul class="role">
+									<li>Back End Dev</li>
+									<li>2015</li>
+								</ul>
+							</div>
+
+						</div>
+						<div class="project-info">
+							<p>
+								A daily selection of privately personalized reads;
+								no accounts or sign-ups required.
+							</p>
+						</div>
+						<div class="project-tag">
+							<ul class="lang">
+								<li>html</li>
+								<li>css</li>
+								<li>javascript</li>
+							</ul>
+						</div>
+					</div>
+					<div class="btn-project">
+						<a href="">See Project</a>
+					</div>
+
+				</div>
+			</div>
+			<div class="details-card">
+				<div class="project-image"><img src="./images/project3.svg" alt="project1img"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Tonic</h1>
```

Comment:
- Kindly, The `h1`  tag indicates a title on a website and it will be one per page , please use `h2`-`h6` based on order, for this one `h2` will be better :+1: